addons/l10n_in_check_gst_status/models/res_partner.py

```diff
@@ -0,0 +1,30 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, _
+from odoo.addons.iap import jsonrpc
+from odoo.exceptions import UserError, AccessError
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_in_gstin_verified_status = fields.Char(string="GSTIN Verified Status", tracking=True)
+    l10n_in_gstin_verified_date = fields.Date(string="GSTIN Verified Date")
+
+    def get_verified_status(self):
+        url = "https://jva-odoo-iap-apps2-15-0-gst-number-active-check-jva-13181640.dev.odoo.com/iap/l10n_in_reports/1/public/search"
```

Comment:
url must not be declared under the methods
May be const? with base_url and end_point separation?